obs-packaging/linux-container/debian.rules

```diff
@@ -19,7 +19,7 @@ override_dh_auto_build:
 	find $(KERNEL_CONFIGS) -name "$(KERNEL_ARCH)_kata_kvm_*" -exec cp {} .config \;
 	[ -f .config ] || (@echo "ERROR: cannot find the kernel config file for the $(KERNEL_ARCH) architecture"; exit 1)
 	make -s ARCH=$(KERNEL_ARCH) oldconfig > /dev/null
-	make -s CONFIG_DEBUG_SECTION_MISMATCH=y ARCH=$(KERNEL_ARCH) -j all
+	make -j "4" -s CONFIG_DEBUG_SECTION_MISMATCH=y ARCH=$(KERNEL_ARCH)
```

Comment:
How about `$(( $(nproc) - 1 ))` ?

/cc @grahamwhaley.

Reply:
The problem is though aiui @jodh-intel , we don't know how many parallel jobs may OOM a builder :-(
I actually think the only safe option here is to use '1' - basically, no `-j` at all (`make` default is single threaded).
As this is an override for packaging only, it should not affect the CI build times I think (where we do care about speed of build, and do have a bit more control over how much resource the builders get).

Reply:
Yep. Maybe we should be using ccache too. It smells a bit odd - presumably we don't have paging enabled but does the OOM point to under-resourced guests I wonder?

Reply:
@amshinde The problem afaiui is that we OOM'd the system with a `-j n`, so I think `-j` on its own will do the same thing - launch as many jobs as it can, and run the system out of memory. Thus, I think the only safe option is no parallel make (no `-j`) - and as this is packaging, I don't think this is a time critical task - so why not go the 'safe route'.